features/admin.console-settings.v1/components/console-administrators/administrators-list/administrators-list.tsx

```diff
@@ -142,12 +165,43 @@ const AdministratorsList: React.FunctionComponent<AdministratorsListProps> = (
         listOffset,
         searchQuery,
         null,
-        selectedUserStore,
+        selectedUserStore ?? selectedAdministratorGroup === "administrators" ? "PRIMARY" : "DEFAULT",
         UserManagementConstants.GROUPS_ATTRIBUTE
     );
 
+    const useOrgConfig: UseOrganizationConfigType = useOrganizationConfigV2;
+
+    const organizationName: string = store.getState().auth.tenantDomain;
+
+    const saasFeatureStatus : FeatureStatus = useCheckFeatureStatus(
+        FeatureGateConstants.SAAS_FEATURES_IDENTIFIER);
+
+    const {
+        data: OrganizationConfig,
+        isLoading: isOrgConfigRequestLoading,
+        isValidating: isOrgConfigRequestRevalidating
+    } = useOrgConfig(
+        organizationName,
+        {
+            revalidateIfStale: true
+        },
+        saasFeatureStatus === FeatureStatus.ENABLED
+    );
+
+    const [ rolesList ] = useState([]);
```

Comment:
Is this an expected state?

Reply:
Fixed with [dd46878](https://github.com/wso2/identity-apps/pull/6938/commits/dd46878991263402cd57a8ede07169b493af93d6)